pydatastructs/graphs/algorithms.py

```diff
@@ -548,3 +550,57 @@ def strongly_connected_components(graph, algorithm):
         "isn't implemented for finding strongly connected components."
         %(algorithm, graph._impl))
     return getattr(algorithms, func)(graph)
+
+
+def random_walk(graph, start, path_length, seed = 0):
```

Comment:
```suggestion
def random_walk(graph, start, path_length, seed=0):
```
No spaces around `=` in arguments of function/method.
In addition, I think doing a random walk is of no use unless we give user an opportunity to perform some operation while the random walk is at a given node. Currently, `breadth_first_search`(see the code in this file) does something like that. May be allowing an argument like operation will also help in implementing loop erased random walks by just defining `operation`.

Reply:
In `breadth_first_search` we have `operation` as a function which is executed for each graph node. While in `random_walk` we can have `operation` of type string where it can take values `uniform` or `loop_erased`. User will give these values and `uniform` will be the default value.
Accordingly the corresponding function will be executed. Will this work ?

Reply:
I don't think that would make the code robust. `operation` should be a function.

Reply:
If operation is a function, I will have to define the random walk function in the test function. Will it be okay?

Reply:
Well, few lines of code are always better than 1000 words of explanation. Please push the commits for showing what you are trying to do.

Reply:
ping @czgdp1807 
I just pushed a commit. For `loop_erased_random_walk` we just need to redefine `operation`.